qiita_pet/handlers/api_proxy/tests/test_studies.py

```diff
@@ -101,5 +102,49 @@ def test_study_prep_get_req_no_access(self):
                'message': 'User does not have access to study'}
         self.assertEqual(obs, exp)
 
+    def test_study_delete_req(self):
```

Comment:
This is modifying the DB - shouldn't the class be decorated with the safeguard that we put in place?